.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
You can add `.vscode` and `.DS_Store` in global git ignore setting. Check out https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/